pipelines/_ingredients/LSSTCam/DRP.yaml

```diff
@@ -108,6 +109,11 @@ tasks:
       python: |
         from lsst.analysis.tools.atools import *
   # These don't actually work now:
+  photometricMatchVisit:
+    class: lsst.analysis.tools.tasks.photometricCatalogMatch.PhotometricCatalogMatchVisitTask
+    config:
+      extraColumns: ["x", "y", "ap09Flux", "ap09FluxErr"]
```

Comment:
Do these have to go here because they're not present when `photometricMatchPreVisit` is run and so would cause that task to fail if they were in `obs_subaru`?

Reply:
If you're referring to the `extraColumns`, I have no opinion on those. They were simply copied in verbatim from `$ANALYSIS_TOOLS_DIR/pipelines/visitQualityCore.yaml`, a YAML which gets imported into this YAML. That makes `photometricMatchVisit` visible in this YAML. I needed to over-ride the `referenceCatalogLoader.doApplyColorTerms` config to `false`, which necessitates me redefining the entire task and prior configs again here.

Reply:
I thought this was the case, but in a recent slack thread there was a big discussion about how the configs are actually supposed to be inherited.